### ISSUE TITLE
Support for Scarab Pricing

### ISSUE THREAD

**Issue by stephenhoran** [klayveR/xenontrade#31]
I attempted to add this support myself however I am unable to really determine the cause of this errors I am receiving. I added the following to parserTypes.json file under the Normal Array:
```
    {
      "regex": "Can be used in the Templar Laboratory or a personal Map Device to add modifiers to a Map.",
      "type": "Scarab"
    }
```

As well as extended the API's in the poe ninja api. 
```
      {overview: "item", type: "Scarab"},
```
However still no luck, just a yellow icon, no errors logging. 

Ninja supports: https://poe.ninja/api/data/itemOverview?league=Betrayal&type=Scarab

I will continue to try and hack on this but maybe I am missing something?

**Comment by klayver** [comment]
You need to add the API url to the poe-ninja-module somewhere. I can't check exactly where right now but I'll update in a day or two.

**Comment by stephenhoran** [comment]
@klayveR Interesting. I assumed that by updating the API's attribute of the NinjaAPI class in the poe-ninja-module would be enough to make the helper able to build the correct url based on type. I will have to see what you do to implement this to better understand.

**Comment by klayver** [comment]
You're right, just checked again and that should work. Maybe the item isn't being parsed correctly or the query to fetch the price needs to be adjusted for scarabs. I currently only have access to a mobile device so I can't check this properly, but will do once I'm back home.

**Comment by stephenhoran** [comment]
@klayveR Yes, very strange. I could not figure out how to log the http request itself as I am pretty new with electron and could not see any http events in the debugger. Thanks for the help! Love the tool, I use it daily!

**Comment by stephenhoran** [comment]
@klayveR it appears this issue has returned in the new league :(
